projects/p11-kit/project.yaml

```
@@ -0,0 +1,3 @@
+homepage: https://p11-glue.github.io/p11-glue/p11-kit.html
```

This should fix the CI error.
```suggestion
homepage: https://p11-glue.github.io/p11-glue/p11-kit.html
main_repo: https://github.com/p11-glue/p11-kit
```

---

Thanks; updated.